Runner/ReadMe.md

```diff
@@ -0,0 +1,45 @@
+
+---------------------------------
+---------------------------------
+------- Batch Labs Runner -------
+---------------------------------
+---------------------------------
+
+This python script is used for testing the Batch Templates inside the BatchExplorer-data\ncj folder. It does not test BatchExplorer only the Azure Batch CLI and its extension client. Please set the following environmental variables before running the job script. 
+
+PS_BATCH_ACCOUNT_NAME
+PS_BATCH_ACCOUNT_KEY
```

Comment:
need this key? and the storage account key bellow? should be able to get it from arm